Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrBuilder.kt

```diff
@@ -146,6 +148,9 @@ class StubsBuildingContextImpl(
 
     private var theCounter = 0
 
+    private val uniqFunctions = mutableSetOf<String>()
+    override fun isOverloading(name: String) = !uniqFunctions.add(name) // TODO params & return type
```

Comment:
Format  is
``
// TODO: sentence with dot at the end.
``

Reply:
Why not to pass `FunctionDecl` directly?